polyfill/test/parseResults.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+
+const { stdin, stdout, exit } = process;
+let testOutput = '';
+const now = Date.now();
+
+stdin.setEncoding('utf8');
+
+stdout.write('Starting Test262 tests.\n');
+
+let testCount = -1;
+let lastReportedCount = -1;
+
+function reportStatus() {
+  const secs = (Date.now() - now) / 1000;
+  stdout.write(`\r${testCount - 1} tests completed in ${secs} seconds. (${Math.round(testCount / secs)} tests/sec)   `);
+}
+
+stdin.on('data', function (chunk) {
+  const lines = chunk.split('\n').length;
```

Comment:
For me locally, this gives a more accurate result:

```suggestion
  const lines = chunk.split('\n').length - 1;
```

but the counting relies heavily on the shape of the JSON serialization. That's probably fine for the process reporting, though.

Reply:
Yep, I was mistakenly doubling the test count. Fixed now.